tendrl_unit_test_setup.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Sets up machine so there can be performed Tendrl unit tests
+# Installs all tendrl packages and pytest
+- hosts: usm_test
```

Comment:
I'm thinking about better/more descriptive name for the `usm_test` group...  what about something like `unittest_node`?